### PR TITLE
Tor 57 build fe of guide page

### PR DESCRIPTION
**Pull request by hippogriffic**

### CODE REVIEW

**Comment by hippogriffic** [comment]
@ababic is working on hooking the cards up but otherwise this should be ready for review

**Comment by ababic** [comment]
Hi @eadpearce. After a bit of digging around, I realised there's an established pattern for `article-detail` URLs that I could utilise here. The only special/temporary thing here is the redirect to capture visits to `/international/how-to-setup-in-the-uk/guides/` and get them to `/international/how-to-setup-in-the-uk/#guides`. This can easily be replaced once we have an actual listing page to sit in between there.

File: conf/urls.py [hunk 1]
```diff
@@ -70,6 +71,16 @@
...
         name="eu-exit-landing"
     ),
+    url(
+        r"^international/how-to-setup-in-the-uk/$",
+        core.views.GuideLandingPageCMSView.as_view(),
+        {'slug': 'how-to-setup-in-the-uk'},
+        name="uk-setup-guides"
+    ),
+    url(
+        r"^international/how-to-setup-in-the-uk/guides/$",
```
Comment (eadpearce) [inline conf/urls.py]: can you add a test for this pls

File: core/templates/core/guide_landing_page.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+	<section class="background-great-blue">
+		<div class="container">
+			<div class="cta-block flex-grid padding-vertical-90">
+				<div class="column-two-thirds">
+					<img class="cta-block-image" src="{{ page.section_two_image.url }}" alt="" width="100%" height="auto">
+				</div>
+
+				<div class="column-third white-text cta-block-content">
+					<h3 class="font-large">{{ page.section_two_heading }}</h3>
```
Comment (eadpearce) [inline core/templates/core/guide_landing_page.html]: this should be Overpass bold so use `heading-large` and remove the top padding from the p below it

**Pull request merged**